lib/Element.js

```diff
@@ -18,6 +14,22 @@ function styleCamelCase (name) {
   }
 }
 
+function isTruthy (value) {
+  return !!value
+}
+
+function isString (value) {
+  return typeof value === 'string'
+}
+
+function isUndefined (value) {
+  return typeof value === 'undefined'
+}
+
+function clone (value) {
+  return Object.assign({}, value)
```

Comment:
I believe this would drop [browser support](http://kangax.github.io/compat-table/es6/#test-Object_static_methods_Object.assign) for IE 11 unless you polyfill it. MDN has a polyfill [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill), however I would just take the polyfill and assign it to your function or stick with loadash's clone. Great idea, loved your article and approach to incorporating mutative libraries with react. I think we're going to give this a shot with an upcoming project.